### PR TITLE
Some cleanup to the HandConstraint* scripts

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change cleans up some unneeded code and improves consistency.

1. Merge Radial and Ulnar/default in the HandConstraintPalmUp switch as they performed idential code.
2. Replace instances of == Handeness with IsHand().  ex: == Handedness.Right -> IsRight()

### CODE REVIEW

**Review by CDiaz-MS — APPROVED** [review]

File: Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs [hunk 1]
```diff
@@ -367,9 +367,6 @@ private bool TryGenerateActivationPoint(IMixedRealityHand jointedHand, out Vecto
                     referenceJoint2 = TrackedHandJoint.Palm;
                     break;
                 case SolverSafeZone.RadialSide:
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs:369]: I think you actually found a bug here! https://github.com/microsoft/MixedRealityToolkit-Unity/issues/8896

Comment (keveleigh) [reply Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs:369]: And going all the way back to the PR, `Ulnar` originally referred to different joints until the syntax was updated and the joints appear to have been copied incorrectly

https://github.com/microsoft/MixedRealityToolkit-Unity/commit/d07969ad624eb8c4682350e02bdf359841c9657d

image

Comment (davidkline-ms) [reply Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs:369]: Hmm.... I will need to check with the original author to see what the intent was.

Comment (davidkline-ms) [reply Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs]: Confirmed with original author. The proper values for UlnarSide should be pinky and wrist. Updated them in the latest commit.

File: Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs [hunk 2]
```diff
@@ -372,8 +372,8 @@ private bool TryGenerateActivationPoint(IMixedRealityHand jointedHand, out Vecto
                     break;
                 case SolverSafeZone.UlnarSide:
                 default:
```
Comment (keveleigh) [inline Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs:374]: This also changes the `default` case. Is that intended?

Comment (davidkline-ms) [reply Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs:374]: yes. the default is ulnar

**Review by keveleigh — APPROVED** [review]

**Pull request merged**